src/hooks/useActiveAccount.tsx

```diff
@@ -19,19 +18,16 @@ export type ActiveAccountProps = {
   account?: Account;
   accountsWithProduct?: Account[];
   accountHeaders?: Record<string, string>;
-  trialExpired?: boolean;
```

Comment:
This `trialExpired` property was not referenced anywhere. Seems like it's fallen out of use.